local-volume/provisioner/Makefile

```diff
@@ -37,18 +28,26 @@ else
 endif
 
 all build:
-	CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' -o local-volume-provisioner ./cmd
+	CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-extldflags "-static"' -o local-volume-provisioner ./cmd
+.PHONY: all build
+
+container: build quick-container
+.PHONY: container
 
-container: build
+quick-container:
 	cp local-volume-provisioner deployment/docker
-	docker build --pull -t $(IMAGE):$(VERSION) deployment/docker
+	docker build -t $(IMAGE):$(VERSION) deployment/docker
```

Comment:
can add a 'docker tag' to ensure :latest is always up-to-date regardless of $(VERSION) https://github.com/kubernetes-incubator/external-storage/blob/master/gluster/block/Makefile#L46

(copied this idea from service-catalog : ))